### ISSUE TITLE
Angular 4 support

### ISSUE THREAD

**Issue by AmirAliSobhGol** [akserg/ng2-slim-loading-bar#43]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => check the FAQ and search github for a similar issue or PR before submitting
[ ] support request => check the FAQ and search github for a similar issue before submitting
[x ] feature request
```
the package seems to be working fine with angular 4, had no issues, if you could just update the dependencies that would be gr8.

**Comment by akserg** [comment]
Hi @AmirAliSobhGol 

I have published the release 4.0.0 on npmjs.com. Please let me know does it work for you?

**Comment by Simon-Briggs** [comment]
Hi, Not OP, but I've been using 4.0.0 for the past 2 weeks with no problems! Thanks.
